src/index.ts

```diff
@@ -377,6 +384,10 @@ class ServerlessAppsyncPlugin {
   async gatherData() {
     const apiId = await this.getApiId();
 
+    if (typeof apiId !== 'string') {
```

Comment:
```suggestion
    if (!apiId) {
```

if for some reason, `apiId` is an empty string, it probably does not make sense to continue either.